### PR TITLE
Expand content on POOL_SIZE in Heroku guide

### PR DESCRIPTION
**Pull request by endersstocker**
Performed small copy edits and used console example for setting POOL_SIZE environment variable for mix task.

### CODE REVIEW

File: deployment/E_heroku.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ Setting config vars and restarting mysterious-meadow-6277... done, v3
 SECRET_KEY_BASE: xvafzY4y01jYuzLm3ecJqo008dVnU3CN4f+MamNd1Zue4pXvfvUjbiXT8akaIF53
 ```
 
-If you need to make any of your config variables available at compile time you will need to explicitly define which ones in a configuration file. Create a file `elixir_buildpack.config` in your applications root directory and add a line like: `config_vars_to_export=(DATABASE_URL MY_VAR)`. See [here](https://github.com/HashNuke/heroku-buildpack-elixir#specifying-config-vars-to-export-at-compile-time) for more information. 
+If you need to make any of your config variables available at compile time you will need to explicitly define which ones in a configuration file. Create a file `elixir_buildpack.config` in your applications root directory and add a line like: `config_vars_to_export=(DATABASE_URL MY_VAR)`. See [here](https://github.com/HashNuke/heroku-buildpack-elixir#specifying-config-vars-to-export-at-compile-time) for more information.
```
Comment (jeregrine) [inline deployment/E_heroku.md]: Could you add `POOL_SIZE` as a default here too. It wasn't part of the original but now that I see it, we're going to need it.

**Comment by jeregrine** [comment]
@endersstocker thank you so much!

File: deployment/E_heroku.md (hunk 1 above)
Comment (endersstocker) [inline deployment/E_heroku.md]: @jeregrine Good call. See cb7db7e.

Comment (endersstocker) [inline deployment/E_heroku.md]: @jeregrine Good call. See 9d3016a.

Comment (endersstocker) [inline deployment/E_heroku.md]: `POOL_SIZE` should only be needed at runtime, not compile time.

File: deployment/E_heroku.md [hunk 2]
```diff
@@ -163,7 +149,7 @@ config :logger, level: :info
 config :hello_phoenix, HelloPhoenix.Repo,
   adapter: Ecto.Adapters.Postgres,
   url: System.get_env("DATABASE_URL"),
-  pool_size: 20,
+  pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
```
Comment (jeregrine) [inline deployment/E_heroku.md]: I believe this variable will need to be available at build time so setting it here https://github.com/phoenixframework/phoenix_guides/pull/514/files#diff-969f42064dcef84e92928f7483e56814R220 will be required.

Comment (endersstocker) [inline deployment/E_heroku.md]: @jeregrine I tested it with and without `config_vars_to_export` in the buildpack configs. It seems that `POOL_SIZE` is only needed at compile time if `|| "10"` is left out. I verified that the Phoenix app actually uses the value of `POOL_SIZE` and not `"10"` in absence of `config_vars_to_export`.

Comment (jeregrine) [inline deployment/E_heroku.md]: Thanks!

**Comment by jeregrine** [comment]
Thank you so much!

**Pull request merged**